### ISSUE TITLE
You are creating too many playlists error

### ISSUE THREAD

**Issue by quyleanh** [linsomniac/spotify_to_ytmusic#75]
I face this issue. How can I fix it? Thank you.
```
ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
You are creating too many playlists. Please wait a while before creating further playlists. in 5 seconds
ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
You are creating too many playlists. Please wait a while before creating further playlists. in 10 seconds
ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
You are creating too many playlists. Please wait a while before creating further playlists. in 20 seconds
ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
You are creating too many playlists. Please wait a while before creating further playlists. in 40 seconds
ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
You are creating too many playlists. Please wait a while before creating further playlists. in 80 seconds
ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
You are creating too many playlists. Please wait a while before creating further playlists. in 160 seconds
```

**Comment by mitexleo** [comment]
> I face this issue. How can I fix it? Thank you.
> 
> ```
> ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
> You are creating too many playlists. Please wait a while before creating further playlists. in 5 seconds
> ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
> You are creating too many playlists. Please wait a while before creating further playlists. in 10 seconds
> ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
> You are creating too many playlists. Please wait a while before creating further playlists. in 20 seconds
> ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
> You are creating too many playlists. Please wait a while before creating further playlists. in 40 seconds
> ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
> You are creating too many playlists. Please wait a while before creating further playlists. in 80 seconds
> ERROR: (Retrying create_playlist: All Out 80s) Server returned HTTP 400: Bad Request.
> You are creating too many playlists. Please wait a while before creating further playlists. in 160 seconds
> ```

You have to wait few hours. I ran the command after waiting more than 3 hours.

**Comment by quyleanh** [comment]
> You have to wait few hours. I ran the command after waiting more than 3 hours.
Yes. But there is no way to automatically check the quota and pass it with sleep?

**Comment by wildstoats** [comment]
I ran into a similar issue. I seems like I had too many playlists to ever successfully copy them to YouTube in a single sitting. To work around this I wrote a little script that splits the `playlists.json` file generated by [spotify-backup.py](https://raw.githubusercontent.com/caseychu/spotify-backup/master/spotify-backup.py) down into smaller chunks and then copied each of the chunks individually.

If you want to give it a go the script is here: https://gist.github.com/wildstoats/386cbdd0d84471c84f036e60bb35da19.

**Comment by david-mcgurrin** [comment]
I was getting this as well and it was duplicating some of my playlists.

A few weeks later my account was permanently banned for [Spam, deceptive practices, & scams policies](https://support.google.com/youtube/answer/2801973?hl=en) - just wondering if anyone else got this and if they could be related!? I'm not saying this tool is responsible but more curious in case it could be linked. Very disappointing anyway.

**Comment by CY83R-3X71NC710N** [comment]
I can confirm your account with get flagged by an automated bot and terminated under the severe spam policy. One way to mitigate this would be running the program from terminal using the --track-sleep=3 option, this would allow the program to not trigger YouTube's antispam detection algorithm.

**Comment by CY83R-3X71NC710N** [comment]
With that said it is now safe to close this issue.

**Comment by lyndonguitar** [comment]
I used the s2yt_copy_all_playlists and encountered this too, and the suggestion here is to wait. However there seems to be a risk for ban too. unless I use  --track-sleep=3, so the command will be "s2yt_copy_all_playlists --track-sleep=3" is that correct?

Also, what if i close the CMD and try again another time. will it automatically recognize the progress I've already made and not re-create the playlists it has already created?

**Comment by CY83R-3X71NC710N** [comment]
I do not believe the program will recognize the progress you've already made, I believe the program will attempt to recreate all the playlists already created, in pull request #109 and my new fork of the project at #117 [https://github.com/CY83R-3X71NC710N/spotify_to_ytmusic]. The program is updated to  append to existing playlists rather than overwrite, however, this will still attempt to add a song and go through every song in the playlist over again. To answer your question about the command, in terminal I run the command ```spotify2ytmusic copy_all_playlists --track-sleep=2 --algo 0```
